birthday/README.md

```diff
@@ -131,7 +129,10 @@ Code:
       {{end}}
    {{end}}
 {{else}}
-   Wait... Deletation is being done.
+   Please wait... deletion is in progress.
```

Comment:
nit picky but `D` here should be capitalized ig

Also, I don't have access to the line on the review, but `Still running deletation..` is also a typo.